library/src/main/java/com/wultra/android/mtokensdk/operation/OperationsService.kt

```diff
@@ -163,6 +186,43 @@ class OperationsService: IOperationsService {
         }
     }
 
+    private fun processServerTime(response: OperationListResponse, requestStarted: ZonedDateTime) {
+
+        val now = ZonedDateTime.now()
+        val requestDelayMilliseconds = now.toInstant().toEpochMilli() - requestStarted.toInstant().toEpochMilli()
+
+        // server does not support this feature
+        if (response.currentTimestamp == null) {
+            return
+        }
```

Comment:
Hope the last note: This should be at the beginning of the function 🥹

Reply:
Yop 👍🏻